contrib/for-tests/volumes-tester/nfs/Dockerfile

```diff
@@ -12,10 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:14.04
+FROM centos
 MAINTAINER Jan Safranek, jsafrane@redhat.com
-ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update -qq && apt-get install -y nfs-kernel-server -qq
+RUN yum -y install /usr/bin/ps nfs-utils && yum clean all
```

Comment:
Sorry, I should have mention it in the log message, rebase to Fedora is needed to have rpc.nfsd with -G option, Ubuntu (14.04) does not compile it in for some reason. I'll update the commit.

Building and pushing was done by @saad-ali, e.g. in #9019.